tests/h/subscribers_test.py

```diff
@@ -219,6 +222,39 @@ def test_it_enqueues_delete_annotation_celery_task_for_delete(
         delete_annotation.delay.assert_called_once_with(event.annotation_id)
         assert not add_annotation.delay.called
 
+    @pytest.mark.parametrize(
+        "action,method",
+        (
+            ("create", "add_annotation_by_id"),
+            ("update", "add_annotation_by_id"),
+            ("delete", "delete_annotation_by_id"),
+        ),
+    )
+    @pytest.mark.usefixtures("with_synchronous_flag")
+    def test_it_calls_sync_service(
+        self, action, pyramid_request, search_index, method, transaction_manager
+    ):
+        event = AnnotationEvent(pyramid_request, {"id": "any"}, action)
+
+        subscribers.sync_annotation(event)
+
+        transaction_manager.__enter__.assert_called_once()
+        getattr(search_index, method).assert_called_once_with(event.annotation_id)
+        transaction_manager.__exit__.assert_called_once()
```

Comment:
```suggestion
        transaction_manager.__enter__.assert_called_once_with()
        getattr(search_index, method).assert_called_once_with(event.annotation_id)
        transaction_manager.__exit__.assert_called_once_with()
```

Make sure they were called with no args?

Reply:
The exit is called with args but it's just internal Python nonsense: it gets called with `None, None, None`. So we could check for that, but it's not actually a property we care about. This is just the easiest way I could find to express the fact we use it as a context manager.